gateway/src/node/client_handling/websocket/connection_handler/authenticated.rs

```diff
@@ -276,7 +290,7 @@ where
 
         self.increase_bandwidth(bandwidth_value as i64).await?;
         let available_total = self.get_available_bandwidth().await?;
-        debug!("Increased bandwidth for client: {:?}", self.client.address);
+        println!("Increased bandwidth for client: {:?}", self.client.address);
```

Comment:
avoid using println, instead use appropriate log macro

Reply:
Yes, I was just removing that but forgot to push